### DISCUSSION TITLE
Realm migration issue (One or more errors occurred. (SQLite Error 1: 'no such column: s.Beatmaps.CountdownOffset'.))

### DISCUSSION THREAD

**Discussion by smileyhead** [ppy/osu#16709] [Q&A] [answered]
I updated the client on Android after having not played on my device for months, and the database migration failed.

Device: Google Pixel 4a
OS: Android 12L Beta 2 (S2B2.211203.006)
[attach_me.zip](https://github.com/ppy/osu/files/7965616/attach_me.zip)

**Comment by peppy** [comment, marked as answer]
Please unzip the attached file to your main osu! directory replacing the existing one to restore your database.

[client.realm.zip](https://github.com/ppy/osu/files/7965624/client.realm.zip)

**Reply by smileyhead** [reply]
Thank you very much!
